schemas/actor.json

```diff
@@ -0,0 +1,171 @@
+                    "description": "Object with parameter-value pairs",
+                    "additionalProperties": true
+                },
+                "required": true
+            },
+            "examples": [
+                {
+                    "name": "Get parameters example",
+                    "params": [
+                        {
+                            "name": "parameters",
+                            "value": ["parameter1", "parameter2"]
+                        }
+                    ],
+                    "result": {
+                        "name": "result",
+                        "value": {
+                            "parameter1": "value_of_parameter1",
+                            "parameter2": "some_other_value"
+                        }
+                    }
+                }
+            ]
+        },
+        {
+            "name": "set_parameters",
+            "params": [
+                {
+                    "name": "parameters",
+                    "schema": {
+                        "type": "object",
+                        "description": "Object with parameter-value pairs",
+                        "additionalProperties": true
+
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            },
+            "examples": [
+                {
+                    "name": "Set parameters example",
+                    "params": [
+                        {
+                            "name": "parameters",
+                            "value": {
+                                "parameter1": "value_of_parameter1",
+                                "parameter2": "some_other_value"
+                            }
+                        }
+                    ],
+                    "result": {
+                        "name": "result",
+                        "value": null
+                    }
+                }
+            ]
+
+        },
+        {
+            "name": "call_action",
+            "description": "Call an Action of this Component. All other arguments are handed to the action.",
+            "params": [
+                {
+                    "name": "action",
+                    "schema": {
+                        "type": "string"
+                    },
+                    "required": true
+                },
+                {
+                    "name": "args",
+                    "schema": {
+                        "type": "array",
+                        "description": "List of positional arguments.",
+                        "items": {
+                            "$ref": "#/components/any"
+                        }
+                    },
+                    "required": false
+                }
+            ],
```

Comment:
Adding a kwarg dictionary option?

Reply:
not sure -- some languages effectively don't have kwargs in a call and rely on position (e.g. C++), so it's safer+simpler with a list.

If there's need later, we could always reconsider/re-evalute.

Reply:
ok